aggregator/handler/config.go

```diff
@@ -113,6 +119,12 @@ func (c *writerConfiguration) NewWriterOptions(
 	if c.MaxBufferSize > 0 {
 		opts = opts.SetMaxBufferSize(c.MaxBufferSize)
 	}
+	if c.IncludeEncodingTime != nil {
```

Comment:
nit: any reason we wouldn't just set the sampling rate to zero if we didn't want to encode

Reply:
In that case you'll just need to set `includeEncodingTime` to false.

Reply:
Yea, what I meant was perhaps it would be easier to get rid of this variable entirely.

Reply:
Sure, was just trying to be explicit about it but yeah can do.

Reply:
I don't feel strongly, feel free to disregard.